src/components/scene/device-motion-permission-ui.js

```diff
@@ -0,0 +1,111 @@
+var registerComponent = require('../../core/component').registerComponent;
+var utils = require('../../utils/');
+var bind = utils.bind;
+
+var constants = require('../../constants/');
+
+var DEVICE_PERMISSION_CLASS = 'a-device-motion-permission';
+var DEVICE_PERMISSION_BTN_CLASS = 'a-device-motion-permission-button';
+
+/**
+ * UI for enabling device motion permission
+ */
+ module.exports.Component = registerComponent('device-motion-permission', {
```

Comment:
It should be called `device-motion-permission-ui.js`

Reply:
Good catch--corrected.

Reply:
Good catch--corrected.